user-docs/latest/content-manager/saving-and-publishing-content.md

```diff
@@ -35,14 +35,12 @@ To unpublish content, click on the **Unpublish** button in the top right corner
 
 ## Deleting content
 
-For collection types, you can delete content by deleting entries of that collection type:
+You can delete content by deleting any entry of a collection type, or the default entry of a single type.
 
-1. In the edit view of the entry, click on the **Delete this entry** button.
+1. In the edit view of the entry, click on the **Delete this entry** button, located on the right of the interface, below the Information box.
```

Comment:
Just to make sure that the "Information box" is a validated term with the Product & Design team.

Reply:
"Information box" (as well as the rest of the wording choices discussed together) validated by the Product & Design team!